src/libraries/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -551,6 +551,11 @@ private async Task LoadIntoBufferAsyncCore(Task serializeToStreamTask, MemoryStr
             }
         }
 
+        ///<summary>
+        /// Creates a <see cref="Stream"/> to read the content from.
+        ///</summary>
+        ///<param name="cancellationToken">The <see cref="CancellationToken"/> to register in case the stream needs to be disposed.</param>
+        ///<returns>The output <see cref="Stream"/>.</returns>
```

Comment:
This is basically a sync version of `CreateContentReadStreamAsync`, so the description can be salvaged from https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpcontent.createcontentreadstreamasync?view=net-5.0 (apart the stuff dealing with `async` and `Task`).